src/components/icons/svgs/CheckmarkIcon.tsx

```diff
@@ -24,47 +19,33 @@ const renderIconSmall = (props: Props) => (
 );
 
 const renderIconLarge = (props: Props) => (
-	<svg
-		height="11"
-		viewBox="0 0 14.986 11.0563"
-		width="15"
-		{...props}
-	>
+	<svg height="11" viewBox="0 0 14.986 11.0563" width="15" {...props}>
 		<path d="M5.5 11.056a1.003 1.003 0 0 1-.71-.297L.29 6.203A1 1 0 1 1 1.71 4.796l3.77 3.818L13.256.316a1 1 0 0 1 1.46 1.367L6.23 10.74a1.003 1.003 0 0 1-.717.316z" />
 	</svg>
 );
 
-const Icon: React.FC<Props> = (props) => (
-	({
-		'l': renderIconLarge(props),
-		's': renderIconSmall(props),
-	} as {[key in NonNullable<Props['size']>]: JSX.Element})[props.size!]
-		// @ts-ignore
-		|| (!console.log(`Invalid prop option passed to switch statement for this icon with props`, props)
-	)
-);
+const Icon: React.FC<Props> = (props) =>
+	((
+		{
+			l: renderIconLarge(props),
+			s: renderIconSmall(props),
+		} as { [key in NonNullable<Props['size']>]: JSX.Element }
+	)[props.size!] ||
+	// @ts-ignore
+	!console.log(`Invalid prop option passed to switch statement for this icon with props`, props));
```

Comment:
Not sure, just linted this file, I'll look into it

Reply:
Okay so seems like this is a way to prevent invalid props from stopping the whole render? Like if I passed `'sm'` instead of `'s'` to this icon's `size` prop, it would still render just an empty icon, because the `!` makes it return true - without it, everything crashes and errors out. Just a way to kind of silently fail with console logging.